msg/DifferentialDriveSetpoint.msg

```diff
@@ -2,3 +2,5 @@ uint64 timestamp # time since system start (microseconds)
 
 float32 speed # [m/s] collective roll-off speed in body x-axis
 float32 yaw_rate # [rad/s] yaw rate
+
+# TOPICS feed_forward_differential_drive_setpoint closed_loop_differential_drive_setpoint
```

Comment:
Shouldn't we name it with postfixes to have them sorted one after the other alphabetically and and a bit shorter to not waste bytes in all logs, listener, ... e.g. something like `differential_drive_setpoint_ff`, `differential_drive_setpoint_cl` ?

Reply:
I did that in a733ec97984b0b417f9b1f235d89581d6038ec46
Hope it makes it more readable also for others.